src/amp/components/Ticker.tsx

```diff
@@ -0,0 +1,94 @@
+import {css} from 'emotion';
+import {headline} from '@guardian/src-foundations/typography';
+import {neutral, brandAlt} from '@guardian/src-foundations/palette';
+import React from 'react';
+
```

Comment:
As a convention its nice to have `React` at the top, and try to separate external libs to internal libs

Reply:
✅